lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -173,18 +173,18 @@ class TestFieldOverrideMongoPerformance(FieldOverridePerformanceTestCase):
 
     TEST_DATA = {
         # (providers, course_width, enable_ccx): # of sql queries, # of mongo queries, # of xblocks
-        ('no_overrides', 1, True): (27, 7, 19),
-        ('no_overrides', 2, True): (135, 7, 131),
-        ('no_overrides', 3, True): (595, 7, 537),
-        ('ccx', 1, True): (27, 7, 47),
-        ('ccx', 2, True): (135, 7, 455),
-        ('ccx', 3, True): (595, 7, 2037),
-        ('no_overrides', 1, False): (27, 7, 19),
-        ('no_overrides', 2, False): (135, 7, 131),
-        ('no_overrides', 3, False): (595, 7, 537),
-        ('ccx', 1, False): (27, 7, 19),
-        ('ccx', 2, False): (135, 7, 131),
-        ('ccx', 3, False): (595, 7, 537),
+        ('no_overrides', 1, True): (26, 7, 19),
+        ('no_overrides', 2, True): (134, 7, 131),
+        ('no_overrides', 3, True): (594, 7, 537),
+        ('ccx', 1, True): (26, 7, 47),
+        ('ccx', 2, True): (134, 7, 455),
+        ('ccx', 3, True): (594, 7, 2037),
+        ('no_overrides', 1, False): (26, 7, 19),
+        ('no_overrides', 2, False): (134, 7, 131),
+        ('no_overrides', 3, False): (594, 7, 537),
+        ('ccx', 1, False): (26, 7, 19),
+        ('ccx', 2, False): (134, 7, 131),
+        ('ccx', 3, False): (594, 7, 537),
```

Comment:
do you know what changed that caused the number of SQL queries to drop?  (it's not bad, i just don't see a corresponding change in the PR)